src/richText.js

```diff
@@ -411,13 +411,15 @@ define([
             replacer, result;
         if (escape) {
             replacer = function () {
-                var id = util.get_guid();
-                places[id] = replacePathWithBubble(form, this.outerHTML);
+                var id = util.get_guid(),
+                    text = xml.humanize($("<div>").append($(this).clone()));
```

Comment:
The value passed to `replacePathWithBubble()` is expected to be a string, and it's not convenient to convert `this` to a string that can easily be `xml.humanize`'d. I had tried `xml.humanize(this.outerHTML)`, but that didn't work (not sure why).